assets/sites/default/settings.php

```diff
@@ -79,7 +79,7 @@ function _data_starter_validates($variable = '') {
 if (isset($conf['default']['fast_file']) && $conf['default']['fast_file']['enable']) {
   $conf['dkan_datastore_fast_import_selection'] = 2;
   $conf['dkan_datastore_fast_import_selection_threshold'] = $conf['default']['fast_file']['limit'];
-  $conf['dkan_datastore_load_data_type'] = 'load_data_local_infile';
+  $conf['dkan_datastore_load_data_type'] = 'load_data_infile';
```

Comment:
@dkinzer I've seen it working on acquia with load_data_infile actually. And I remember we having problems with load_data_local_infile because we needed to pass the --extra="--local-infile=0" to the mysql daemon.